resources/composer.json

```diff
@@ -3,8 +3,9 @@
     "guzzlehttp/guzzle": "^6.2.1"
   },
   "require-dev": {
-    "behat/behat": "~3.1.0@rc",
-    "drupal/drupal-extension": "~3.1.0",
+    "behat/behat": "~3.1",
+    "drupal/drupal-extension": "~3.1",
```

Comment:
Generates lots of notices which make the build to fail.
To be handled on another PR.